go.mod

```diff
@@ -5,7 +5,7 @@ go 1.18
 require (
 	cosmossdk.io/math v1.0.0-beta.3
 	github.com/armon/go-metrics v0.4.1
-	github.com/cosmos/cosmos-sdk v0.46.5-0.20221111074045-68e54fa5b868
+	github.com/cosmos/cosmos-sdk v0.46.6
```

Comment:
can you post the diff? 🙏

Reply:
https://github.com/cosmos/cosmos-sdk/compare/v0.46.5...v0.46.6

Reply:
should be all the commit from `68e54fa5b868` to last right?
https://github.com/cosmos/cosmos-sdk/commits/v0.46.6


anyways I can see a breaking change in group module but I assume it does not affect us

Reply:
the gov fix potentially affects us / is breaking, I think, but I don't know if there were any new gov proposals executed on the testnet?

Reply:
oh yes, it's from the commit: https://github.com/cosmos/cosmos-sdk/compare/68e54fa5b868...v0.46.6

Reply:
I think the gov change only affect us if we had a proposal execution returned an error.